### DISCUSSION TITLE
Connection method USB cable can not used

### DISCUSSION THREAD

**Discussion by kxs109** [pawelsalawa/sqlitestudio#4173] [Q&A]
Hello, somebody who can help me?

adb: adb version
Android Debug Bridge version 1.0.41
Version 30.0.5-6877874

![11627213049_ pic_hd](https://user-images.githubusercontent.com/13159216/126897884-4f78514a-95e2-457b-a8c6-43f7eb0063cc.jpg)

![31627213284_ pic](https://user-images.githubusercontent.com/13159216/126897916-f28e8ee3-507e-41dc-b64d-4b81f51d914f.jpg)

**Comment by pawelsalawa** [comment]
SQLiteStudio is unable to locate your ADB automatically. Click this: 
![image](https://user-images.githubusercontent.com/481846/126952952-c07add02-6deb-4b66-9673-79021bd487a3.png)
to point out where your adb executable is.

**Reply by kxs109** [reply]
@pawelsalawa thank you very much!
